bin/ch-builder2squash

```diff
@@ -36,18 +36,25 @@ if [ "$#" -lt 2 ]; then
 fi
 
 builder_choose
-echo "builder: ${CH_BUILDER}"
 
 image=$1
 outdir=$2
 shift 2
 
+if [ ! -e "$outdir" ]; then
```

Comment:
I'd recommend `-d` here, to be more specific. You might then change the error message to something along the lines of "${outdir} is not a directory".